nixos/modules/services/backup/borgbackup.md

```diff
@@ -52,8 +52,8 @@ because the specified keys are restricted to running borg serve and can only
 access this single repository. You need the output of the generate pub file.
 
 ```ShellSession
-# sudo ssh-keygen -N '' -t ed25519 -f /run/keys/id_ed25519_my_borg_repo
-# cat /run/keys/id_ed25519_my_borg_repo
```

Comment:
That doesn't even work. If you run `ssh-keygen` as root, it will create key with `0600` permissions, so you can't `cat` it as regular user.